test/tests/functional/pbs_test_tpp.py

```diff
@@ -44,15 +44,21 @@ class TestTPP(TestFunctional):
     """
     Test suite consists of tests to check the functionality of pbs_comm daemon
     """
+    node_list = []
 
     def setUp(self):
         TestFunctional.setUp(self)
         self.pbs_conf = self.du.parse_pbs_config(self.server.shortname)
         self.exec_path = os.path.join(self.pbs_conf['PBS_EXEC'], "bin")
+        a = {'resources_available.ncpus': 2}
+        for mom in self.moms.values():
+            self.server.manager(MGR_CMD_SET, NODE, a, id=mom.shortname)
```

Comment:
looking at the test cases, you are not using more than 2 ncpus anywhere. Any particular reason for setting ncpus=2 here?

Reply:
In the test we are submitting reservation which is spanning across 2 nodes requesting 1 ncpus. Also a job is submitted into the reservation queue which would too span across 2 nodes and requests 1 ncpus. The test is then expecting job to be in R state. So total 4(2+2) ncpus are required

Reply:
jobs submitted to reservation queue utilise resources from the reservation itself